app/controllers/application_controller.rb

```diff
@@ -39,6 +41,14 @@ def request_host_include?(hosts)
     hosts.any? { |host| Addressable::URI.parse(request.referer).host.include? host } rescue false
   end
 
+  def has_comments?
+    comments_count > 0
+  end
+
+  def comments_count
+    Assignment.where(submitter_id: current_user.id).try(:flat_map, &:comments).try(:reject, &:readed).count
+  end
+
```

Comment:
May this be a performance issue in the future? 

Perhaps we could simply add an association with `:through` option from user to comments. 

Then we could do `current_user.try(:comments).try(:reject, &:read).count`